op-node/withdrawals/utils.go

```diff
@@ -264,23 +267,17 @@ func ParseWithdrawalInitiated(receipt *types.Receipt) (*bindings.L2ToL1MessagePa
 	if err != nil {
 		return nil, err
 	}
-	abi, err := bindings.L2ToL1MessagePasserMetaData.GetAbi()
-	if err != nil {
-		return nil, err
-	}
 
 	for _, log := range receipt.Logs {
-		event, err := abi.EventByID(log.Topics[0])
-		if err != nil {
```

Comment:
I handled this problem like this:

https://github.com/ethereum-optimism/optimism/blob/5a5e6ef15ee272af99da20e1b3a5716c97a26dee/op-chain-ops/crossdomain/legacy_withdrawal_test.go#L216-L218

Where when the abi didn't know about the event, it would return `nil` for the event. 

Its unlikely that the event will change so i think its fine to hardcode the topic

Reply:
This suppresses the error returned by `abi.EventByID`. It'll return an `event not found` error when it encounters a topic it doesn't know about.